src/app/components/transaction/in-microblock-label.tsx

```diff
@@ -0,0 +1,35 @@
+import { FiInfo } from 'react-icons/fi';
+
+import { Box, Flex, Stack, Text, color } from '@stacks/ui';
+import { SendFormSelectors } from '@tests-legacy/page-objects/send-form.selectors';
+
+import { Tooltip } from '@app/components/tooltip';
+
+const inMicroblockMessage =
+  'This transaction is currently in a microblock, which increases the chances of inclusion in the next anchor block.';
```

Comment:
Where is this text coming from? `toolTipLabel`?

Reply:
Mark helped me write that message out